scripts/generate_errors.pl

```diff
@@ -90,6 +90,9 @@
     $include_name =~ tr/A-Z/a-z/;
     $include_name = "" if ($include_name eq "asn1");
 
+    # Fix faulty ones
+    $include_name = "net_sockets" if ($module_name eq "NET");
```

Comment:
The source file `library/error.c` is (at least) partly auto generated, since the module name `NET` no longer matches the actual include name `net_sockets.h` we need to manually deal with this case in the generator script.